astropy/cosmology/io/latex.py

```diff
@@ -57,7 +57,7 @@ def write_latex(
     table = to_table(cosmology, cls=cls, cosmology_in_meta=False)
 
     cosmo_cls = type(cosmology)
-    for name, col in table.columns.copy().items():
+    for name in tuple(table.colnames):
```

Comment:
Wouldn't this be even faster?:
```suggestion
    for name in table.columns:
```

Reply:
Is that safe, like from the problem in https://stackoverflow.com/questions/6777485/modifying-a-python-dict-while-iterating-over-it ?

Reply:
Reverted.

Reply:
Iterating over a `dict` iterates over its keys, so changing the keys while iterating is a bad idea, but changing the values of a `dict` does not affect the keys, so that is safe. In this case the keys of `table.columns` are the column names, but the for-loop only attaches a unit to the columns where appropriate, i.e. it modifies the values, not the keys.